src/views/Contacts.vue

```diff
@@ -662,6 +667,32 @@ export default {
 			this.downloadVcardPromise(vcardPromise)
 		},
 
+		/**
+		 * Open mailto: for contacts in a group
+		 *
+		 * @param {Object} group of contacts to be emailed
+		 */
+		emailGroup(group) {
+		  const emails = []
+			group.contacts.forEach(key => {
+				// The email property could contain "John Doe <john.doe@example.com>", but vcard spec only
+				// allows addr-spec, not name-addr, so to stay compliant, replace everything outside of <>
+				const email = this.contacts[key].email.replace(/(.*<)([^>]*)(>)/g, '$2').trim()
+				const name = this.contacts[key].fullName.replace(/[,<>]/g, '').trim()
+				if (email === '') {
+					return
+				}
+				if (name === null || name === '') {
+					emails.push(email)
+					return
+				}
+				emails.push(`${name} <${email}>`)
+			})
+			// We could just do mailto:${emails}, but if we want to use name-addr, not addr-spec, then we
+			// have to explicitly set the "To:" header. Plus, we want to BCC the recipients
+			window.location.href = `mailto:?bcc=${emails.map(encodeURIComponent).join(',')}`
```

Comment:
Super tiny enhancement: A space after the comma would be nice here to separate the different recipients better. Doesn’t matter for the actual functionality, just looks a bit nicer while writing. :)

Reply:
I did try this and I think I found that one of the clients would then add %20 before each of the email addresses. It was a while ago now, so I'll have to double check

Reply:
Ahh ok, then nevermind! :)